org_fedora_oscap/rule_handling.py

```diff
@@ -895,25 +907,26 @@ def __str__(self):
     def eval_rules(self, ksdata, storage, report_only=False):
         """:see: RuleHandler.eval_rules"""
 
+        firewall_proxy = NETWORK.get_proxy(FIREWALL)
         messages = []
 
         if self._firewall_default_enabled is None:
             # firewall default startup setting
-            self._firewall_default_enabled = ksdata.firewall.enabled
+            self._firewall_default_enabled = firewall_proxy.FirewallMode
```

Comment:
I would suggest to call this variable `_firewall_detault_state`, the current name is confusing.

Reply:
Or `_firewall_default_mode`.